add new keyboard atset (#17195)
qmk/qmk_firmware: keyboards/atset/at9/readme.md

# ATSET AT9

> !atset/at9 

Reviewer: This needs to be an image.

*A SPECIAL 9 KEY KEYBOARD POWERED BY QMK*

* Keyboard Maintainer: [ATSET](https://github.com/%USER_NAME%)

> * Hardware Supported: *The PCBs, ATMEGA 32u4*

Reviewer: Please fill these out.

Make example for this keyboard (after setting up your build environment):

>     make %AT9%:default
> 
> Flashing example for this keyboard:
> 
>     make %AT9%:default:flash

Reviewer: ```suggestion
    make atset/at9:default

Flashing example for this keyboard:

    make atset/at9:default:flash
```

> *A SPECIAL 9 KEY KEYBOARD POWERED BY QMK.HAVING SPECIAL LAYER FEATURES WHICH CAN BE PROGRAMMED AS PER YOUR CONVENIENCE *
> 
> * Keyboard Maintainer: [ATSET](https://github.com/%anubhavd7%)
> * Hardware Supported: * ATMEGA 32u4*
> 
> Make example for this keyboard (after setting up your build environment):
> 
>      make atset/at9:default
>      
> Flashing example for this keyboard:
> 
>     make atset/at9:default:flash

Reviewer: ```suggestion
A special 9 key keyboard powered by QMK.

* Keyboard Maintainer: [ATSET](https://github.com/anubhavd7)
* Hardware Supported: ATmega32u4

Make example for this keyboard (after setting up your build environment):

     make atset/at9:default
     
Flashing example for this keyboard:

    make atset/at9:default:flash

```

# ATSET AT9

> ![AT9](imgur.com image replace me!)
> 
> *A SPECIAL 9 KEY KEYBOARD POWERED BY QMK*
> 
> * Keyboard Maintainer: [ATSET](https://github.com/%USER_NAME%)
> * Hardware Supported: *The PCBs, ATMEGA 32u4*
> 
> Make example for this keyboard (after setting up your build environment):
> 
>     make %AT9%:default
> 
> Flashing example for this keyboard:
> 
>     make %AT9%:default:flash

Reviewer: ```suggestion
![AT9](imgur.com image replace me!)

*A SPECIAL 9 KEY KEYBOARD POWERED BY QMK*

* Keyboard Maintainer: [ATSET](https://github.com/%USER_NAME%)
* Hardware Supported: *The PCBs, ATMEGA 32u4*

Make example for this keyboard (after setting up your build environment):

    make atset/at9:default

Flashing example for this keyboard:

    make atset/at9:default:flash
```

Please fill these out.
